includes/functions/functions_strings.php

```diff
@@ -49,10 +49,10 @@ function zen_preserve_search_quotes(?string $search_string): string
  * with parameters that run htmlspecialchars over the string
  * and converts quotes to html entities
  *
- * @param string The string to be parsed
+ * @param  string  $string  The string to be parsed
```

Comment:
I'm not objecting to keeping this, but since the comment is redundant now that the type is in the function signature, this `@param` line can just be deleted.
Similar throughout the rest of the file.

In cases where the function sig type is *not* declared in the function, then we do want it in the docblock as the cue for IDEs and code-analyzers to aid in dev hints/warnings.

Reply:
I was thinking more of these lines for parameters being places to add information/comments.